toolset/setup/linux/client.sh

```diff
@@ -21,9 +21,9 @@ sudo sh -c "echo '*               -    nofile          65535' >> /etc/security/l
 # wrk
 ##############################
 
-git clone https://github.com/wg/wrk.git
-cd wrk
-git checkout 205a1960c8b8de5f500bb143863ae293456b7add
+curl -sL -o wrk-4.0.1.tar.gz https://github.com/wg/wrk/archive/4.0.1.tar.gz
```

Comment:
Please undo the changes to this file so that it is not included in your pull request.